.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Enhancement:
+
+Reason:
+
+Result:
+
+Issue Tracker Tickets (Jira or BZ if any):
```

Comment:
IMHO, this is the format for RFE (new feature). And for bugfix, we usually use a different format in bugzilla:
```
Cause: 

Consequence: 

Fix: 

Result: 
```

Reply:
> IMHO, this is the format for RFE (new feature). And for bugfix, we usually use a different format in bugzilla:
> 
> ```
> Cause: 
> 
> Consequence: 
> 
> Fix: 
> 
> Result: 
> ```

We don't have a good way for users to select a specific template, and there isn't an automated way to do it.  So no matter what we do, the template isn't going to be applicable for all issues, and reviewers will need to guide users in how best to fill out these.  Note that the goal is that we can use these directly in BZ Doc Text without having to go back weeks or months after the fact and fill that in.

See also the discussion in https://github.com/linux-system-roles/ssh/pull/99

Reply:
Ok, then I am fine with this template, thanks.